components/dashboard/src/projects/NewProject.tsx

```diff
@@ -193,33 +218,40 @@ export default function NewProject() {
-                    </div>
-                </ContextMenu>
-                <div className="w-full relative ">
-                    <img src={search} title="Search" className="filter-grayscale absolute top-1/3 left-3" />
-                    <input className="w-96 pl-10 border-0" type="text" placeholder="Search Repositories" value={repoSearchFilter}
-                        onChange={(e) => setRepoSearchFilter(e.target.value)}></input>
+        const renderRepos = () => (<>
+            <div className={`mt-10 border ${isGitHub() ? "rounded-t-xl border-b-0" : "rounded-xl"} border-gray-100 flex-col`}>
+                <div className="px-8 pt-8 flex flex-col space-y-2">
+                    <ContextMenu classes="w-full left-0 cursor-pointer" menuEntries={getDropDownEntries(accounts)}>
+                        <div className="w-full">
+                            {icon && (
+                                <img src={icon} className="rounded-full w-6 h-6 absolute top-1/4 left-4" />
+                            )}
+                            <input className="w-full px-12 cursor-pointer font-semibold" readOnly type="text" value={selectedAccount || ""}></input>
+                            <img src={CaretDown} title="Select Account" className="filter-grayscale absolute top-1/2 right-3" />
+                        </div>
+                    </ContextMenu>
+                    {filteredRepos.length > 0 && (
+                        <div className="w-full relative ">
+                            <img src={search} title="Search" className="filter-grayscale absolute top-1/3 left-3" />
+                            <input className="w-96 pl-10 border-0" type="text" placeholder="Search Repositories" value={repoSearchFilter}
+                                onChange={(e) => setRepoSearchFilter(e.target.value)}></input>
+                        </div>
+                    )}
                 </div>
-            </div>
-            <div className="p-6 flex-col">
-                {reposToRender.length > 0 && (
-                    <div className="overscroll-contain max-h-80 overflow-y-auto pr-2">
-                        {reposToRender.map(r => (
-                            <div key={`repo-${r.name}`} className="flex p-3 rounded-xl hover:bg-gray-100 dark:hover:bg-gray-800 focus:bg-gitpod-kumquat-light transition ease-in-out group" >
-
-                                <div className="flex-grow">
-                                    <div className="text-base text-gray-900 dark:text-gray-50 font-medium rounded-xl whitespace-nowrap">{toSimpleName(r.name)}</div>
-                                    <p>Updated {moment(r.updatedAt).fromNow()}</p>
-                                </div>
+                <div className="p-6 flex-col">
+                    {filteredRepos.length > 0 && (
+                        <div className="overscroll-contain max-h-80 overflow-y-auto pr-2">
```

Comment:
**suggestion**: What do you think of using a smaller max height like `max-h-52` here? Suggesting this mostly because the scolling area is overflowing below the viewport height in some cases. Your call! We can also pick up such details in https://github.com/gitpod-io/gitpod/issues/4948 and https://github.com/gitpod-io/gitpod/issues/4951.

| BEFORE | AFTER |
|-|-|
| <img width="1440" alt="Screenshot 2021-08-19 at 5 31 45 PM" src="https://user-images.githubusercontent.com/120486/130087367-47e4b4c1-0408-46d6-83d2-45a53a0ed110.png"> | <img width="1440" alt="Screenshot 2021-08-19 at 5 31 23 PM" src="https://user-images.githubusercontent.com/120486/130087376-6737a4cf-750d-4065-a011-11beb61485c1.png"> |

Reply:
**fyi**: Opened follow up issue in https://github.com/gitpod-io/gitpod/issues/5334.